Anima/diffusion/mcm_estimator/animaMCMEstimatorImageFilter.hxx

```diff
@@ -549,11 +564,16 @@ MCMEstimatorImageFilter<InputPixelType, OutputPixelType>
     {
         this->InitialOrientationsEstimation(mcmOptimizationValue,currentNumberOfCompartments,initialDTI,observedSignals,
                                             generator,threadId,aiccValue,b0Value,sigmaSqValue);
-
-        this->TrunkModelEstimation(mcmOptimizationValue,observedSignals,threadId,aiccValue,b0Value,sigmaSqValue);
-        if ((m_CompartmentType != Stick)&&(m_CompartmentType != Zeppelin))
+        
+        if (m_CompartmentType == NODDI)
```

Comment:
If SpecificModelEstimation is not so much useful anymore, I could think of refactoring the code differently. To see with what you are doing on DDI

Reply:
Well there is the coarse grid init search for NODDI that I intend to use also for DDI (and since the parameters have the same ranges and interpretation, I guess a single function could fit both models. Tensor estimation is done a bit differently at the moment since we are generating pseudo-random numbers from Halton sequences. I guess we could also use a coarse grid search approach here which would accelerate tremendously tensor estimation.